50-main.config

```diff
@@ -41,14 +41,22 @@ GLIDEIN_ResourceName = "Docker"
 
 GLIDEIN_ContainerTag = "@CONTAINER_TAG@"
 
-STARTD_ATTRS = $(STARTD_ATTRS) GLIDEIN_Country GLIDEIN_Site GLIDEIN_ResourceName WorkerGroupName IsOsgVoContainer GLIDEIN_ContainerTag
+STARTD_ATTRS = $(STARTD_ATTRS) GLIDEIN_Country GLIDEIN_Site GLIDEIN_ResourceName \
+               WorkerGroupName IsOsgVoContainer GLIDEIN_ContainerTag \
+               IsBlackHole HasExcessiveLoad
+
+IsBlackHole = IfThenElse(RecentJobDurationAvg is undefined, false, RecentJobDurationCount >= 10 && RecentJobDurationAvg < 180)
+HasExcessiveLoad = LoadAvg > 2*DetectedCpus + 2
 
 # only accept new work for a while, for validated nodes and jobs with project name set
 # On GPU nodes, CPU only jobs are only allowed after all GPUs are in use
 START = (isUndefined(DaemonStartTime) || (time() - DaemonStartTime) < (MY.ACCEPT_JOBS_FOR_HOURS*60*60)) && \
-        (MY.OSG_NODE_VALIDATED == True) && \
+        (isUndefined(MY.OSG_NODE_VALIDATED) || MY.OSG_NODE_VALIDATED) && \
```

Comment:
This also runs jobs if OSG_NODE_VALIDATED is undefined - is that on purpose? When would OSG_NODE_VALIDATED be undefined?

Reply:
The main concern would be other VOs picking up the container and maybe using different validation scripts.

Reply:
Can you add a comment saying that, so I don't ask this question next time?

Reply:
Done